src/itemsTotal.js

```diff
@@ -0,0 +1,10 @@
+import { getData } from './api.js';
+
+const itemTotal = async () => {
+  const total = await getData();
+  const items = document.querySelector('.items');
+  items.textContent = `Total Disney Characters: ${total.length} Characters`;
```

Comment:
- Hey, please, kindly note that the total number of characters on your page now is 100 and not 50, after you fix those eslint export issues, it would be the correct value as 50. Please, kindly watch out for this while updating the codebase.

Reply:
Done